src/foam/u2/detail/SectionedDetailPropertyView.js

```diff
@@ -250,59 +236,72 @@ foam.CLASS({
 
           return self.E()
             .addClass(this.myClass('wrapper'))
-            .start(self.Rows)
-              .callIf(prop$label, function() {
-                this.start('m3')
-                  .add(prop.label)
-                  .style({ 'line-height': '2' })
-                .end();
-              })
-              .start()
-                .style({ 'position': 'relative', 'display': 'inline-flex', 'width': '100%' })
-                .start()
-                
-                  .style({ 'flex-grow': 1, 'max-width': '100%' })
-                  .callIf( prop.view.class == 'foam.u2.CheckBox', function() {
-                    this.addClass(this.myClass('CheckBoxWrapper'));
-                  })
-                  .tag(prop, { mode$: self.mode$ })
-                  .callIf(prop.validationStyleEnabled, function() {
-                    this.enableClass(self.myClass('error'), errorSlot);
-                  })
-                .end()
-                .callIf(prop.help, function() {
-                  this.start('', { tooltip: prop.help })
-                    .start({class: 'foam.u2.tag.Image', data: '/images/question-icon.svg'})
-                      .addClass(self.myClass('helper-icon'))
-                    .end()
+            .start()
+              .start(self.Rows)
+                .callIf(prop$label, function() {
+                  this.start('m3')
+                    .add(prop.label)
+                    .style({ 'line-height': '2' })
                   .end();
                 })
-              .end()
-              .callIf(prop.validationTextVisible && ( mode === self.DisplayMode.RW || mode === self.DisplayMode.DISABLED ), function() {
-                this
+                .start()
+                  .style({ 'position': 'relative', 'display': 'flex', 'flex-wrap': 'wrap', 'width': '100%' })
                   .start()
-                    .style({ 'align-items': 'center' })
-                    .start(self.Cols)
-                      .addClass(self.myClass('validation-container'))
-                      .show(errorSlot)
-                      .start({
-                        class: 'foam.u2.tag.Image',
-                        data: '/images/inline-error-icon.svg',
-                        displayHeight: '16px',
-                        displayWidth: '16px'
-                      })
-                        .style({
-                          'justify-content': 'flex-start',
-                          'margin': '0 8px 0 0'
+                    .style({ 'flex-grow': 1, 'max-width': '100%' })
+                    .callIf( prop.view.class == 'foam.u2.CheckBox', function() {
+                      this.addClass(this.myClass('CheckBoxWrapper'));
```

Comment:
Why is this needed?

Reply:
Used to align checkboxes and labels(in case they are present).

Reply:
Couldn't I just do the same with a better CSS selector instead of adding ChecBoxWrapper?

Reply:
True, also can't think of any reason for this to live outside the CheckBox View.